extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj

```diff
@@ -6,7 +6,8 @@
     <Description>ASP.NET Core extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>1.3.3</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionSuffix>-preview1</VersionSuffix>
```

Comment:
Unrelated to this change. But should we pin to specific versions other package references below (Worker.Extensions.Abstraction etc.,) ?

Reply:
This shouldn't be required, but I would like to follow up on that separately. Let's get the proper version in place and follow up.